cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -43,13 +42,29 @@ func generateFile(gen *protogen.Plugin, file *protogen.File) *protogen.Generated
 	filename := file.GeneratedFilenamePrefix + "_grpc.pb.go"
 	g := gen.NewGeneratedFile(filename, file.GoImportPath)
 	g.P("// Code generated by protoc-gen-go-grpc. DO NOT EDIT.")
+	g.P("// versions:")
+	g.P("// \tprotoc-gen-go-grpc v", version)
+	g.P("// \tprotoc             v", protocVersion(gen))
```

Comment:
Let's replace the `\t` with `- ` instead.

Reply:
I simply copied what protoc-gen-go does to match its header, are you sure about this?

Reply:
Yes, I noticed that.  Please change it, thank you.